pkg/function/copy_volume_data.go

```diff
@@ -133,11 +136,24 @@ func (*copyVolumeDataFunc) Exec(ctx context.Context, tp param.TemplateParams, ar
 	if err = OptArg(args, CopyVolumeDataEncryptionKeyArg, &encryptionKey, restic.GeneratePassword()); err != nil {
 		return nil, err
 	}
+	if err = OptArg(args, CopyVolumeDataPodOverrideArg, &podOverride, tp.PodOverride); err != nil {
```

Comment:
You might want to move this all into a helper function in the function pkg - possibly in `args.go`. It'll simplify the repeated code in the functions.

e.g.

```
// GetPodSpecOverride 
func GetPodSpecOverride(tp param.TemplateParams, args map[string]interface{}, argName string) (podOverride map[string]interface{}, err error)
```